components/compositing/compositor.rs

```diff
@@ -1669,6 +1669,7 @@ impl IOCompositor {
                 }],
             );
             self.send_scroll_positions_to_layout_for_pipeline(&pipeline_id);
+            self.dispatch_scroll_window_event_class(pipeline_id, scroll_tree_node);
```

Comment:
```suggestion
            self.dispatch_scroll_event(pipeline_id, scroll_tree_node);
```

I'm not sure where the "event_class" naming comes from but it doesn't really make sense any longer. I appreciate you trying to match the old code though.